lms/templates/courseware/course_about.html

```diff
@@ -240,22 +240,21 @@ <h1>
           % endif
             ## We plan to ditch end_date (which is not stored in course metadata),
             ## but for backwards compatibility, show about/end_date blob if it exists.
-            % if get_course_about_section(request, course, "end_date") or course.end:
-                <%
-                    course_end_date = course.end
-                %>
-
+            <%
+               course_end_date = get_course_about_section(request, course, "end_date") or course.end
+            %>
+            % if course_end_date:
             <li class="important-dates-item">
                 <span class="icon fa fa-calendar" aria-hidden="true"></span>
                 <p class="important-dates-item-title">${_("Classes End")}</p>
-                  % if isinstance(course_end_date, str):
-                      <span class="important-dates-item-text final-date">${course_end_date}</span>
-                  % else:
-                    <%
-                        course_date_string = course_end_date.strftime('%Y-%m-%dT%H:%M:%S%z')
-                    %>
-                    <span class="important-dates-item-text final-date localized_datetime" data-format="shortDate" data-datetime="${course_date_string}"></span>
-                  % endif
+                    % if isinstance(course_end_date, (str, unicode)):
```

Comment:
Unless you've seen instances of errors of end_dates being unicode, I don't think this is a necessary check.

Reply:
I believe `course.end` is either be a properly formatted date string, or `None`...